client/tests/conftest.py

```diff
@@ -149,24 +149,24 @@ def s3(aws_credentials):
 
 @pytest.fixture(scope="module")
 def local_provider_source():
-        def get_local():
-            ff.register_user("test_user").make_default_owner()
-            provider = ff.register_local()
-            source = provider.register_file(
-                name="transactions",
-                variant="quickstart",
-                description="A dataset of fraudulent transactions.",
-                path=f"{dir_path}/test_files/input_files/transactions.csv"
-            )
-            return (provider, source)
-        
-        return get_local
+    def get_local(is_insecure):
```

Comment:
Is this parameter used?

Reply:
It's not, but given the hosted fixture requires it, I had to pass it to the local counterpart. Is there an idiomatic way of ignoring a parameter in Python?

Reply:
It is. I needed to pass a param to the local fixture given the hosted fixture requires it. I'll convert this to an `_` to better convey the fact I don't intend to use it.